src/types/serverless.ts

```diff
@@ -19,6 +19,7 @@ export type VariableResolver = {
         options: Record<string, string>;
     }) => { value: string | Record<string, unknown> } | Promise<{ value: string | Record<string, unknown> }>;
 };
+export type DeprecatedVariableResolver = (variable: string) => Promise<Record<string, unknown>>;
```

Comment:
does it need to be async ?
```suggestion
export type DeprecatedVariableResolver = (variable: string) => Record<string, unknown>;
```

Reply:
Yep